scripts/train.py

```diff
@@ -0,0 +1,41 @@
+import os
+import click
+import pickle
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
+              default=True)
+@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
```

Comment:
@rstevanak I personally think this would look better to be multichoice option rather then switch argument. Something like `@click.option('--data', type=click.Choice(['PKLot', 'PKSpace']))` or `@click.option('--loader', type=click.Choice(['PKLot', 'PKSpace']))`, but I guess it is just a matter of taste. But there has to definitely be a description to these parameters. Without it, one might think if he specifies this parameter we will load data automatically which is not true, this just specifies loader of the data. That is also why I think one option might be better here.